src/WebJobs.Script/Config/ExtensionBundleOptions.cs

```diff
@@ -2,14 +2,21 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using NuGet.Versioning;
 
-namespace Microsoft.Azure.WebJobs.Script.BindingExtensionBundle
+namespace Microsoft.Azure.WebJobs.Script.Configuration
 {
     public class ExtensionBundleOptions
     {
         public string Id { get; set; }
 
         public VersionRange Version { get; set; }
+
+        public ICollection<string> ProbingPaths { get; set; }
+
+        public string DownloadPath { get; set; }
+
+        public bool EnsureLatest { get; set; }
```

Comment:
What is the purpose of EnsureLatest? should we default to the latest version available in the bundles?

Reply:
Setting this would get the latest extension bundle within the given version range. Not the absolute latest

Reply:
Can you give an example? As users already pick a version range, we should just default to latest version we have downloaded on to the machines.

Reply:
Discussed with @pragnagopa offline, We should use / download the latest version of bundle available to us. App service should already have the latest version before CDN so download wouldn't kick in for App service environment.

Reply:
Going back and looking at hosting environments and release process. I am reverting to download extension bundle only when a matching bundle is not already present on vm or user storage.